test/acceptance/test/ui_tests.go

```diff
@@ -75,16 +80,199 @@ var _ = Describe("Weave GitOps UI Test", func() {
 		Expect(webDriver.Destroy()).To(Succeed())
 	})
 
-	It("SmokeTest - Verify gitops can run UI without apps installed", func() {
+	It("UITest - Verify gitops can add apps from the UI to an empty cluster", func() {
+		var addAppPage *pages.AddAppPageElements
+		var repoAbsolutePath string
+		tip := generateTestInputs()
+		appName := tip.appRepoName
+		private := true
+		appRepoRemoteURL := "https://github.com/" + GITHUB_ORG + "/" + tip.appRepoName + ".git"
 
-		dashboardPage := pages.Dashboard(webDriver)
-		expectedTitle := "Weave GitOps"
+		dashboardPage = pages.GetDashboardPageElements(webDriver)
+		addAppPage = pages.GetAddAppPageElements(webDriver)
 
-		By("Then I should be able to navigate to WeGO dashboard", func() {
-			Expect(webDriver.Navigate(WEGO_UI_URL)).To(Succeed())
-			str, _ := webDriver.Title()
-			Eventually(str, THIRTY_SECOND_TIMEOUT).Should(ContainSubstring(expectedTitle))
-			Expect(dashboardPage.ApplicationTab).Should(BeFound())
+		defer deleteRepo(tip.appRepoName, gitproviders.GitProviderGitHub, GITHUB_ORG)
+		defer deleteWorkload(tip.workloadName, tip.workloadNamespace)
+
+		By("When I navigate to Add Application page", func() {
+			Expect(dashboardPage.AddAppButton.Click()).To(Succeed())
+		})
+
+		By("Then I should see Add Applcation page", func() {
+			Expect(addAppPage.AddAppHeader.Text()).Should(ContainSubstring(addApplicationPageHeader))
```

Comment:
Suggestion: can use MatchText matcher
Eventually(addAppPage.AddAppHeader).Should(MatchText(".*addApplicationPageHeader.*"))

Reply:
will address it on the next PR